src/components/ImageLayout.js

```diff
@@ -1,17 +1,28 @@
-import React from 'react';
-import { connect } from 'react-redux'
+import React, {useState} from 'react';
+import {connect} from 'react-redux'
 import Image from './Image.js'
 import CommentLayout from './CommentLayout'
 import * as TYPES from '../store/actions'
 
 export const ImageLayout = ({
-  onDoubleClick,
   image,
-  onKeyDown,
   deleteComment,
   updateCommentEditing,
   editComment,
+  addComment
 }) => {
+  const [inputBoxText, setInputBoxText] = useState("");
+
+  const onChange = (e) => {
+    setInputBoxText(e.target.value)
+  }
+  const onKeyDown = (e, imageId) => {
+    if (e.keyCode === 13) {
+      const value = e.target.value
+      addComment(value, imageId)
+      setInputBoxText("")
+    }
+  }
```

Comment:
looks like none of these are being tested?

Reply:
now they are :)